# Service to update parameter name and value of the vimba camera:
string param_name
string param_val
---
bool successfully_set

